.github/workflows/ci.yml

```diff
@@ -67,6 +67,10 @@ jobs:
             python: "2.7"
     steps:
       - uses: actions/checkout@v2.3.1
+      # Save about 10 minutes off of the test time
+      - name: Configure source-static to not test the devel role (for PR CI)
```

Comment:
will it be tested in another scenario?

Reply:
Whoops! This was not supposed to be included in the PR. It is incomplete too.